Tasks/SqlDacpacDeploymentOnMachineGroup/ExecuteScriptsInTransaction.sql

```diff
@@ -0,0 +1,282 @@
+PRINT 'Begin Sql scripts execution'
+SET XACT_ABORT ON
+SET NOCOUNT ON
+
+DECLARE @_acquireLock VARCHAR(64)
+DECLARE @_acquireLockMilliseconds INT
+DECLARE @_acquireLockMaxAttempts INT
+DECLARE @_longRunningThresholdMilliSeconds INT
+DECLARE @_lastAttemptsToKillSessions INT
+DECLARE @_isHosted INT
+
+PRINT 'Adding SQL files into temporary table _tmpSqlFilesTable'
+
+IF OBJECT_ID (N'_tmpSqlFilesTable', N'U') IS NOT NULL 
+BEGIN
+   PRINT 'Temporary table exists already. Dropping table _tmpSqlFilesTable'
+   DROP TABLE _tmpSqlFilesTable
+END
+CREATE TABLE _tmpSqlFilesTable (ScriptIndex INT NOT NULL IDENTITY, SqlScriptContent NVARCHAR(MAX) NULL)
```

Comment:
We are trying to do schema changes from Stored Procedure executions. That task may fail with security errors - if user provides restricted access